config/main.py

```diff
@@ -1667,6 +1673,65 @@ def load_port_config(config_db, port_config_path):
                 port_name), display_cmd=True)
     return
 
+
+def load_golden_config(golden_config_path):
+    if not os.path.isfile(golden_config_path):
+        return
+
+    # Override configDB with golden config
+    clicommon.run_command('config override-config-table {}'.format(
```

Comment:
>config [](http://example.com/codeflow?start=27&length=6)

This function is already inside `config`, so no need to start another process, you can call a python function.